dts/bindings/sensor/invensense,icm42605.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2020 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: ICM-42605 motion tracking device
+
+# ICM-42605 is SPI.
+compatible: "invensense,icm42605"
+
+include: spi-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        The INT signal default configuration is active-high.  The
+        property value should ensure the flags properly describe the
+        signal that is presented to the driver.
+
+    cpha:
+      type: boolean
+      required: true
+      description: |
+        Set to indicate phase starts with asserted half-phase (CPHA=1).
+        For this driver using this property requires also using cpol.
+
+    cpol:
+      type: boolean
+      required: true
+      description: |
+        Set to indicate clock leading edge is falling (CPOL=1).
+        For this driver using this property requires also using cpha.
```

Comment:
cpha/cpol, if these are `boolean` and `required` why do we need properties for them?

Reply:
As you commented, cpha/cpol is already set on code side.
I followed other SPI driver and I also agree about cpha cpol is no need on yaml side.

        drv_data->spi_cfg.operation = (SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
                        SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE |
                        SPI_TRANSFER_MSB);

Will delete them in yaml

Reply:
@galak Removed